packages/unity-bootstrap-theme/src/scss/extends/_image-text-block.scss

```diff
@@ -12,10 +12,6 @@
       flex-direction: row-reverse;
     }
 
-    @include media-breakpoint-up(md) {
-      max-height: 540px;
```

Comment:
@mlsamuelson After the meeting with UI, I think this might need to be added back in